Dolapo47/Auto-Mart
# refactor validation done on all routes in the app [finish #166973946]

#### What does this PR do?
- A summary of what this task is about
this PR creates validation for the 4 routes in the app

#### Description of Task to be completed?
- A list of the major tasks you did to complete this feature/bug/chore
installed Joi as a dependency in the app
used Joi to validate user input
#### How should this be manually tested?
- Outlined steps on how someone else can successfully test this feature/chore/bug to ensure it works as expected
clone app to local directory
run npm install to get the app's dependencies
npm start to run the app
input incorrect user details, you will get the respective error message
#### What are the relevant pivotal tracker stories?
- PT Story link - (format - `[#166973946](https://www.pivotaltracker.com/n/projects/2346179)`)
